pkg/output/table_filter.go

```diff
@@ -0,0 +1,88 @@
+package output
+
+import (
+	"fmt"
+	"text/tabwriter"
+)
+
+func CheckIfColumnIsEmpty(columnIdx int, tableData [][]string) bool {
+	for _, row := range tableData {
+		if columnIdx < len(row) && row[columnIdx] != "" {
+			return false
+		}
+	}
+	return true
+}
+
+func RemoveEmptyColumns(headers []string, tableData [][]string) [][]string {
+	if len(tableData) == 0 {
+		return [][]string{headers}
+	}
+
+	var newHeaders []string
+	var columnsToKeep []int
+
+	for i, header := range headers {
+		if !CheckIfColumnIsEmpty(i, tableData) {
+			newHeaders = append(newHeaders, header)
+			columnsToKeep = append(columnsToKeep, i)
+		}
+	}
+
+	var result [][]string
+	result = append(result, newHeaders)
+
+	for _, row := range tableData {
+		var newRow []string
+		for _, colIdx := range columnsToKeep {
+			if colIdx < len(row) {
+				newRow = append(newRow, row[colIdx])
+			} else {
```

Comment:
`colIdx` -> `col` (lets keep naming consistent..)

Reply:
fixed in last commit